From 99bbdc042ee4c92a13edcfeaf336aa755a199d0f Mon Sep 17 00:00:00 2001
From: Anup Patel <anup@brainfault.org>
Date: Sun, 11 Sep 2016 19:53:40 +0530
Subject: [PATCH] Makefile: Fix oldconfig make target

Use "-s" instead of "-o" option of "openconf/conf" tool because
the "-o" option does not generate all required openconf files.

Signed-off-by: Anup Patel <anup@brainfault.org>
Signed-off-by: Eric Le Bihan <eric.le.bihan.dev@free.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 149a682..e05e801 100644
--- a/Makefile
+++ b/Makefile
@@ -434,7 +434,7 @@ menuconfig:
 oldconfig:
 	$(V)mkdir -p $(OPENCONF_TMPDIR)
 	$(V)$(MAKE) -C tools/openconf oldconfig
-	./tools/openconf/conf -o $(OPENCONF_INPUT)
+	./tools/openconf/conf -s $(OPENCONF_INPUT)
 
 # Rule for "make savedefconfig"
 .PHONY: savedefconfig
-- 
2.4.11

